lib/common/options.c

```diff
@@ -640,12 +639,11 @@ pcmk__format_option_metadata(const char *name, const char *desc_short,
 {
     /* big enough to hold "pacemaker-schedulerd metadata" output */
     GString *s = g_string_sized_new(13000);
-    int lpc = 0;
 
-    g_string_append_printf(s, "<?xml version=\"1.0\"?>"
-                              "<resource-agent name=\"%s\">\n"
-                              "  <version>%s</version>\n",
-                              name, PCMK_OCF_VERSION);
+    pcmk__g_strcat(s,
+                   "<?xml version=\"1.0\"?>"
```

Comment:
This should end with a newline

Reply:
Good catch

Reply:
Oh, that one didn't end in a newline before the change either. I wondered about whether that was intentional or not. I'll still add one as long as it doesn't have any side effects.